src/postgraphql/__tests__/pathgraphqlIntegrationSchemaExport-test.js

```diff
@@ -0,0 +1,28 @@
+jest.mock('fs')
+
+import withPgClient from '../../postgres/__tests__/fixtures/withPgClient'
+import createPostGraphQLSchema from '../schema/createPostGraphQLSchema'
+import exportPostGraphQLSchema from '../schema/exportPostGraphQLSchema'
+import { writeFileSync } from 'fs'
+
+test('exports a schema as JSON', withPgClient(async pgClient => {
+  const gqlSchema = await createPostGraphQLSchema(pgClient, ['a', 'b', 'c'])
+  await exportPostGraphQLSchema(gqlSchema, { exportPathJSON: '/schema.json' })
```

Comment:
Unfortunately, increasing the timeout does not seem to help. I went as far as 3 times the value used in the tests you linked, and it still times out.

When I do not mock `fs`, the timeout does not occur, and I get an error about not being able to read property `calls` on undefined, which is expected as the `mocks` property will be undefined.

Additionally, when I do mock `fs`, it seems to never even reach the `createPostGraphQLSchema` call. If I add a logging statement right above it, it never is called.

Any ideas?